synopsys-task/task.json

```diff
@@ -227,6 +228,71 @@
       "visibleRule": "polarisReportsSarifCreate = true",
       "groupName": "scanOptions"
     },
+    {
+      "name": "polarisAzureToken",
+      "type": "string",
+      "label": "Azure Token",
+      "defaultValue": "",
+      "required": false,
+      "helpMarkDown": "PAT or System.AccessToken has to be passed with legitimate access when Coverity/BlackDuck/Polaris PR comment and BlackDuck FIX PR is enabled.",
```

Comment:
We should only mention about Polaris here since this section is Polaris Specific but Coverity/BlackDuck is also mentioned.

Correct message should be - `PAT or System.AccessToken has to be passed with legitimate access when Polaris PR comment is enabled.`